test/unit/auth/credential.spec.ts

```diff
@@ -395,6 +395,36 @@ describe('Credential', () => {
         privateKey: mockCertificateObject.private_key,
       });
     });
+
+    it('should parse valid token if app def creds point to default refresh token loc', () => {
+      process.env.GOOGLE_APPLICATION_CREDENTIALS = GCLOUD_CREDENTIAL_PATH;
+
+      let readFileSyncStub;
```

Comment:
There's an `fsStub` variable already in scope for these tests. See if we can use that here.